libraries/pod/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! Crate containing `Pod` types and `bytemuck` utils used in SPL
+
+pub mod non_zero_pubkeys;
+pub mod pod_slice;
```

Comment:
nit: maybe we call this module just `slice` since none of the others have `pod` in the name?